File: jdbc_gem_scala/pipelines/FirstPipeline/code/src/main/scala/io/prophecy/pipelines/firstpipeline/graph/target_7.scala
package io.prophecy.pipelines.firstpipeline.graph

import io.prophecy.libs._
import io.prophecy.pipelines.firstpipeline.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object target_7 {

  def apply(context: Context, in: DataFrame): Unit = {
    val Config = context.config
    var writer = in.write.format("jdbc")
    writer = writer
      .option("url",      Config.JDBC_URL)
      .option("dbtable",  "test_table_destination")
      .option("user",     sys.env(s"${Config.ENV_JDBC_USERNAME}"))
      .option("password", sys.env("JDBC_PASSWORD"))
      .option("driver",   Config.DRIVER_NAME)
    writer = writer.mode("overwrite")
    writer.save()
  }

}

File: pythonGemMigration/pipelines/Pipeline_first/code/pipeline_first/graph/target_7.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.utils import *
from prophecy.libs import typed_lit
from pipeline_first.config.ConfigStore import *
from pipeline_first.udfs.UDFs import *

def target_7(spark: SparkSession, in0: DataFrame):
    in0.write\
        .format("jdbc")\
        .option("url", f"{Config.JDBC}/test_database")\
        .option("dbtable", f"{Config.TEST}_table_destination")\
        .option("user", "{}".format(f"{Config.TEST}_{Config.USER}"))\
        .option("password", "{}".format(f"{Config.ADM}{Config.IN}"))\
        .option("driver", Config.DRIVER_NAME)\
        .mode("overwrite")\
        .save()
